common/src/main/java/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -280,7 +291,11 @@ class Backend(
         onSuccessHandler: () -> Unit,
         onErrorHandler: (PurchasesError) -> Unit
     ) {
-        enqueue(object : Dispatcher.AsyncCall() {
+        val cacheKey = listOfNotNull(
+            appUserID,
```

Comment:
looks like tests only check that we don't repeat the call if both are the same. maybe we can add a test to confirm that if EITHER parameter has changed, we WILL repeat the call?

Reply:
Sure, that's a good catch. Want me to write it or you take care of it?

Reply:
i can!